crates/nu-cli/Cargo.toml

```diff
@@ -23,17 +23,17 @@ nu-color-config = { path = "../nu-color-config", version = "0.70.1"  }
 reedline = { version = "0.13.0", features = ["bashisms", "sqlite"]}
 
 atty = "0.2.14"
-chrono = "0.4.21"
+chrono = { default-features = false, features = ["std"], version = "0.4.21" }
 crossterm = "0.24.0"
 fancy-regex = "0.10.0"
 fuzzy-matcher = "0.3.7"
 is_executable = "1.0.1"
 lazy_static = "1.4.0"
 log = "0.4"
 miette = { version = "5.1.0", features = ["fancy-no-backtrace"] }
-percent-encoding = "2"
+percent-encoding = { default-features = false, version = "2" }
```

Comment:
If I understand correctly, you are disabling the `alloc` feature in the `percent-encoding` crate: https://github.com/servo/rust-url/blob/master/percent_encoding/Cargo.toml

I don't understand why; Nushell is always going to be operating in an environment where memory allocation is OK.

Reply:
That is a dependency-less single file crate and the primary operation we currently use seems to just use implementations that work on `&str` and `&[u8]` or are `Iterator` so that just reduces the availability of a few currently unused functions but on the other hand probably does not provide that much compile time benefit (no change in the dependency tree, few lines, no funky generics)